spec/js_routes/options_spec.rb

```diff
@@ -13,6 +13,21 @@
   let(:_options) { {} }
   let(:_warnings) { true }
 
+  context "when serializer is specified" do
+    let(:_options) { {:serializer => "myCustomSerializer"} }
+
+    it "should set configurable serializer" do
+      # define custom serializer
+      # this is a nonsense serializer, which always returns foo=bar
+      # for all inputs
+      evaljs(%q(function myCustomSerializer(object, prefix) { return "foo=bar"; }))
+
+      # expect the nonsense serializer above to have appened foo=bar
+      # to the end of the path
+      expect(evaljs(%q(Routes.inboxes_path()))).to include("?foo=bar")
```

Comment:
updated this to

    expect(evaljs(%q(Routes.inboxes_path()))).to eql("/inboxes?foo=bar")